### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Copy Card Payload] - Screen Reader remain silent when user is activating the 'Copy Card Payload' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8361]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/designer/
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Copy Card Payload' button and press enter key and observe the issue.

**Actual Result:**
Screen Reader is not conveying the any information when user is activating the 'Copy Card Payload' button. It is remain silent.

**Expected Result:**
After activating the 'Copy Card Payload' button, screen reader should convey the information about the activated control and it should announce as **'Copy Card Payload copied'**.

**Observation:**
Same issue repro through the NVDA Screen Reader.

**User Impact:**
It will impact the screen reader user if the screen reader remain silent as users who are dependent on the screen reader will not sure whether control has been activated or not?

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/95913748/224643332-f9c2a15a-21dc-4641-a81d-5ee1ba3a43e8.mp4

![MAS1 3 1-Screen reader is not conveying any information after activating copy card payload button](https://user-images.githubusercontent.com/95913748/224643354-5fcb592b-695d-49bb-91ab-711b0e9277fa.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

Test Environment:
OS Version: 23H2 (Build 25931.1000)
Browser: Edge Dev (108.0.2060.1)
URL: https://adaptivecards.io/designer/
Screen reader: Narrator
